config/initializers/sidekiq.rb

```diff
@@ -36,12 +36,41 @@ def find_free_redis_db(redis_client, max_db = 15)
   raise "No free Redis databases available."
 end
 
+def store_master_schedule_config
```

Comment:
I think we should handle the case where we have switch a site from a proxy to a master and vise-versa